src/external/libjson/CMakeLists.txt

```diff
@@ -16,11 +15,6 @@ SOURCES
   Source/JSONWorker.cpp
   Source/JSONAllocator.cpp
   Source/JSONIterators.cpp
-COMPILE_FLAGS_EXTRA
-  -Wno-reorder
-COMPILE_FLAGS_EXTRA_GCC
-  -Wno-cpp
-COMPILE_FLAGS_EXTRA_CLANG
-  -Wno-header-guard
-  -Wno-\#warnings
 )
+
+target_compile_options(libjson PRIVATE -w)
```

Comment:
Also going to be that guy: let's append the newline here (and in other instances below), so that any future addendums to this file don't necessarily mutate this line.